share/completions/krita.fish

```diff
@@ -28,6 +28,11 @@ YCbCrA' $current_token
     end
 end
 
+function __krita_list_workspaces
+    ls --format single-column --color=never $HOME/.local/share/krita/workspaces/ |
```

Comment:
Please don't parse `ls` output, and don't mess with gnu-specific options.

Also the `string replace` doesn't use `-f` / `--filter`, so any files not called `.kws` will just be passed through as-is.

This should be:

```fish
path basename ~/.local/share/krita/workspaces/*.kws | path change-extension ''
```